### PR TITLE
test: Labels.tsx

### PR DESCRIPTION
**Pull request by SilviaZeta**
## Description
add tests for entering a new label and deleting and re-adding previous labels.

addresses #412

## Dependency changes
yes, adds packages required for React Testing Library to work (same Jest settings as CURATE, see `package.json`).

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] New migrations have been committed
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] If appropriate, I have bumped any version numbers

### CODE REVIEW

File: src/annotation/index.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { Toolbox, Toolboxes } from "@/Toolboxes";
+import { Toolbox, Toolboxes } from "../Toolboxes";
```
Comment (cooper667) [inline src/annotation/index.ts:1]: Why this change?

**Review by cooper667 — DISMISSED** [review]

File: src/annotation/index.ts (hunk 1 above)
Comment (SilviaZeta) [reply src/annotation/index.ts:1]: I changed that path, because Jest didn't find the module when using the alias and I didn't want to venture into config territory.

**Review by cooper667 — DISMISSED** [review]

**Review by ChasNelson1990 — APPROVED** [review]

**Comment by SilviaZeta** [comment]
the last commit is just a merge. pushed with the wrong commit message.

**Review by cooper667 — APPROVED** [review]

**Review by ChasNelson1990 — APPROVED** [review]
